### PR TITLE
Add docs for M6

### PR DESCRIPTION
**Pull request by Zhou-Duo**
Converted the M6 handout to markdown format.
Todo:
- Update the links to "Final Paper & Poster"
- Incorporate the FAQ section

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Benchmarking & Characterization
+
+The [handout describing each team&#39;s paper report](https://www.google.com/url?q=https://docs.google.com/document/d/e/2PACX-1vRZNbg9T3aHVi5B_IcRnpPYhOKpIOylOYF3FbIh06ABxLLZzmnbhuad-ZWnc1BtPciRJKdxoDdlUxLs/pub&sa=D&source=editors&ust=1712121655486555&usg=AOvVaw3pLbHeIVij-y0uB_aeWHs6) contains [an extensive section on benchmarking and characterization](https://www.google.com/url?q=https://docs.google.com/document/d/e/2PACX-1vRZNbg9T3aHVi5B_IcRnpPYhOKpIOylOYF3FbIh06ABxLLZzmnbhuad-ZWnc1BtPciRJKdxoDdlUxLs/pub%23h.my7rmzey516h&sa=D&source=editors&ust=1712121655486673&usg=AOvVaw2cMB0ACtjc1kWVGp0k5kxv)of your system as well as ideas for figures and plots visualizing the results. Note that you must pick appropriate benchmarks for your project and justify them — for example, for a search engine that targets patents it is important to characterize the size and complexity of the input as well as any additional processing for storing and querying these patents.
```
Comment (Zhou-Duo) [inline doc/README.md:75]: We should also update these two links after also converting the handout for the "Final Paper & Poster"

File: doc/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+- build early but functional prototypes of all components — e.g., single node indexer, base page-rank, etc. — because tuning the final ranking function will take time.
+- use version control, integrate and deploy regularly / continuously, and test both individual components and the end-to-end integration.
+- make sure to test your implementation locally—and do so frequently, possibly on every commit—before deploying to AWS
+- keep a close eye on the AWS dashboard and appropriately pause/resume EC2 instances based on their intended use and phase of the project.
+- focus on the core functionality first — only then worry about optimizations and beautification; a beautiful search engine that shows no results has little value.
+
+
+## Reflections & The Way Forward
+
+Remember to keep notes of the technical challenges, the solutions used by your system, and your own reflections — all these will be important while you are [working on your report](https://www.google.com/url?q=https://docs.google.com/document/d/e/2PACX-1vRZNbg9T3aHVi5B_IcRnpPYhOKpIOylOYF3FbIh06ABxLLZzmnbhuad-ZWnc1BtPciRJKdxoDdlUxLs/pub&sa=D&source=editors&ust=1712121655488433&usg=AOvVaw0el7ybdYZjQ_P-Q_TLHRva).
```
Comment (Zhou-Duo) [inline doc/README.md:111]: same as above

File: doc/README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Project Submission
+
+Please include a URL to your repository *in your final paper report*. The repository must include a *fully reproducible version of the results reported in the paper* — including the entire codebase, benchmarks, and results on the set of machines used to produce these results.
+
+
```
Comment (Zhou-Duo) [inline doc/README.md:118]: We will add the FAQ section here once we receive feedback and questions later during M6.

**Review by nvasilakis — APPROVED** [review]
This looks great, thanks so much!

**Pull request merged**